{{cookiecutter.repostory_name}}/app/envs/prod/celery-entrypoint.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
```

Comment:
I think you should quote the original author of this script. What license do they use?

Reply:
They use "gist.github.com" license :D But you're right, I rewrote the code using only my own head and now I'm the owner >:)